scripts/step0_rename_fasta_desc.py

```diff
@@ -1,4 +1,28 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
+
+import argparse
+import os
+
+parser = argparse.ArgumentParser(description="Rename FASTA sequence names")
+
+parser.add_argument(
+    "-i", "--input", required=True, help="path to input FASTA file", action="store"
+)
+parser.add_argument(
+    "-o",
+    "--output",
+    required=True,
+    help="path to output renamed FASTA file",
+    action="store",
+)
+parser.add_argument(
+    "-s",
+    "--sample",
+    required=True,
+    help="sample name to append to sequence name",
```

Comment:
You should give an example of the expected input, i.e. a string with no spaces probably 

`sample1,sample2,sample3`

You'll need to parse these into a list below